File: app/src/main/java/com/semye/android/webview/jsbridge/DefaultHandler.java
package com.semye.android.webview.jsbridge;

/**
 * 默认的handler处理的实现
 */
public class DefaultHandler implements BridgeHandler {

    String TAG = "DefaultHandler";

    @Override
    public void handler(String data, CallBackFunction function) {
        if (function != null) {
            function.onCallBack("DefaultHandler response data");
        }
    }

}

File: app/src/main/java/com/semye/android/webview/jsbridge/DefaultHandler.kt
package com.semye.android.webview.jsbridge

/**
 * 默认的handler处理的实现
 */
class DefaultHandler : BridgeHandler {
    var TAG = "DefaultHandler"
    override fun handler(data: String?, function: CallBackFunction?) {
        function?.onCallBack("DefaultHandler response data")
    }
}